en/intro_to_command_line/README.md

```diff
@@ -397,16 +397,17 @@ Cool, huh? :)
 
  Here is a summary of some useful commands:
 
-Command (Windows) | Command (Mac OS / Linux) | Description | Example
-------------- | ----- | ---------------------- | ---------------------------------------------
-exit          | exit  | close the window       | **exit**
-cd            | cd    | change directory       | **cd test**
-dir           | ls    | list directories/files | **dir**
-copy          | cp    | copy file              | **copy c:\test\test.txt c:\windows\test.txt**
-move          | mv    | move file              | **move c:\test\test.txt c:\windows\test.txt**
-mkdir         | mkdir | create a new directory | **mkdir testdirectory**
-rmdir (or del)| rm    | delete a file          | **del c:\test\test.txt**
-rmdir /S      | rm -r | delete a directory     | **rm -r testdirectory**
+Command (Windows) | Command (Mac OS / Linux) | Description                | Example
+----------------- | ------------------------ | -------------------------- | ---------------------------------------------
+exit              | exit                     | close the window           | **exit**
+cd                | cd                       | change directory           | **cd test**
+cd                | pwd                      | show the current directory | **cd** (Windows) or **pwd** (Mac OS / Linux)
```

Comment:
I think that's okay for now -- maybe we can defer to coach here to explain nitty-gritty details like that :)